### PR TITLE
Change border size to be consistent and change to using BorderFactory - Highscore plugin

### PR DESCRIPTION
**Pull request by TwisterMonkey**

### CODE REVIEW

**Comment by Nightfirecat** [comment]
What is the motivation behind this change? Does it work as expected? If so, why does this work, and the previous border layout not work? (see the deleted comment)

**Comment by TwisterMonkey** [comment]
The old code does exactly what it's meant to do (create an 18-pixel thick border on the top side). Couldn't replicate the bug from the deleted comment on either Windows or Linux.

**Comment by raiyni** [comment]
To provide context for this issue, the comment was fixed in

https://github.com/runelite/runelite/commit/7426babd3f779cb342e028869831e9cca8cd55e8#diff-c0894a1affffbc117d3f2d0cb069b3ec06d01e0ab16ef144e1ed1529a4da6017R168

File: runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java [hunk 1]
```diff
@@ -43,10 +43,7 @@
 import java.util.Map;
 import javax.annotation.Nullable;
 import javax.inject.Inject;
-import javax.swing.ImageIcon;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
-import javax.swing.SwingUtilities;
+import javax.swing.*;
```
Comment (raiyni) [inline runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java:46]: Undo star import.

File: runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java [hunk 2]
```diff
@@ -145,10 +142,7 @@ public HiscorePanel(@Nullable Client client, HiscorePlugin plugin, HiscoreConfig
 		this.nameAutocompleter = nameAutocompleter;
 		this.hiscoreClient = new HiscoreClient(okHttpClient);
 
-		// The layout seems to be ignoring the top margin and only gives it
-		// a 2-3 pixel margin, so I set the value to 18 to compensate
-		// TODO: Figure out why this layout is ignoring most of the top margin
-		setBorder(new EmptyBorder(18, 10, 0, 10));
+		setBorder(BorderFactory.createEmptyBorder(10, 10, 0, 10));
```
Comment (raiyni) [inline runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java:145]: `BorderFactory.createEmptyBorder` is just `new EmptyBorder`

**Review by raiyni — CHANGES_REQUESTED** [review]

**Pull request merged**